### PR TITLE
prac 6 review

### PR DESCRIPTION
**Pull request by AlbertRequilme**
Please review my code @ChenShuxiao20
Thank you!

### CODE REVIEW

File: prac_06/car.py
Comment (ChenShuxiao20) [inline prac_06/car.py:1]: Good

File: prac_06/guitar.py
Comment (ChenShuxiao20) [inline prac_06/guitar.py:1]: Good clear docstring

File: prac_06/guitars.py
Comment (ChenShuxiao20) [inline prac_06/guitars.py:1]: Overall, it's good, but there are some minor problems.

File: prac_06/guitar_test.py
Comment (ChenShuxiao20) [inline prac_06/guitar_test.py:1]: The output is different from the sample output.

**Review by ChenShuxiao20 — COMMENTED** [review]
Good

**Comment by AlbertRequilme** [comment]
Thank you for the review.

**Pull request closed without merging**